Community Highlights, 7th edition (#15788)
home-assistant/home-assistant.github.io: source/_posts/2020-11-30-community-highlights.markdown

---
title: "Community Highlights: 7th edition"
description: "A 'Dad status' board, illuminating the steps of your stairs and the most beautiful thermostat you've ever seen"
date: 2020-11-30 00:00:00
date_formatted: "November 30, 2020"
author: Franck Nijhof
author_twitter: frenck
categories: Community
og_image: /images/blog/2020-11-30-community-highlights/social.png
---

The 7th edition of the Home Assistant Community Highlights! Some interesting
things popped up around our community, we thought was worth sharing.

Troy Hunt has jumped into our wonderful world and shared his journey
in a blog post series. Some nice examples on a DIY solution for mounting a
wall tablet and how to arm your alarm with a toothbrush!

## Troy Hunt is unraveling IoT using Home Assistant

If you have never heard of Troy Hunt before, he is a web security specialist

> and mostly for his ["Have I Been Pwned"][hibp] creation.

Reviewer: ```suggestion
and mostly known for creating ["Have I Been Pwned"][hibp] ([Home Assistant integration](/integration/haveibeenpwned/).
```

> Troy has been diving into the world of IoT and started automating his home,
> using... Home Assistant! In 5 blogs, he writes about his journey in the big
> world of IoT:

Reviewer: ```suggestion
Troy has been diving into the world of IoT and started automating his home,
using... Home Assistant!

> _"I'm massively impressed with HA and nothing else I've seen along my IoT journey comes even close to comparing. If you're going to do IoT in any meaningful way, you start with HA."_ – Troy Hunt

In 5 blogs, he writes about his journey in the big
world of IoT:
```

---
title: "Community Highlights: 7th edition"
description: "A 'Dad status' board, illuminating the steps of your stairs and the most beautiful thermostat you've ever seen"
date: 2020-11-30 00:00:00
date_formatted: "November 30, 2020"
author: Franck Nijhof
author_twitter: frenck
categories: Community
og_image: /images/blog/2020-11-30-community-highlights/social.png
---

The 7th edition of the Home Assistant Community Highlights! Some interesting
things popped up around our community, we thought was worth sharing.

Troy Hunt has jumped into our wonderful world and shared his journey
in a blog post series. Some nice examples on a DIY solution for mounting a
wall tablet and how to arm your alarm with a toothbrush!

## Troy Hunt is unraveling IoT using Home Assistant

If you have never heard of Troy Hunt before, he is a web security specialist
and mostly known for creating ["Have I Been Pwned"][hibp] ([Home Assistant integration](/integration/haveibeenpwned/)).

Troy has been diving into the world of IoT and started automating his home,
using... Home Assistant!

> > _"I'm massively impressed with HA and nothing else I've seen along my IoT journey comes even close to comparing. If you're going to do IoT in any meaningful way, you start with HA."_ – Troy Hunt

Reviewer: Forgot our blockquote automatically adds quotes

<img width="86" alt="image" src="https://user-images.githubusercontent.com/1444314/100712985-ed665000-33b3-11eb-9f55-e0d2ca3da15c.png">

```suggestion

> I'm massively impressed with HA and nothing else I've seen along my IoT journey comes even close to comparing. If you're going to do IoT in any meaningful way, you start with HA.
```

[stl]: https://www.thingiverse.com/thing:2853298

## Arm your alarm by brushing your teeth...

Automatically arm your alarm, when Home Assistant detects you are
using your toothbrush... Why? Because we can! 😎

<blockquote class="twitter-tweet">
<p lang="en" dir="ltr">Have you ever seen a house alarm system controlled by a toothbrush? Here our <a href="https://twitter.com/VanderbiltInd?ref_src=twsrc%5Etfw">@VanderbiltInd</a> alarm is integrated into <a href="https://twitter.com/home_assistant?ref_src=twsrc%5Etfw">@home_assistant</a> . Our toothbrush is ble enabled, it’s state is read by an ESP32 chip and sent into <a href="https://twitter.com/home_assistant?ref_src=twsrc%5Etfw">@home_assistant</a> <a href="https://t.co/T9XGi1Zjmb">pic.twitter.com/T9XGi1Zjmb</a></p>&mdash; Ian (@imduffy15) <a href="https://twitter.com/imduffy15/status/1329569523855478784?ref_src=twsrc%5Etfw">November 19, 2020</a>
</blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

## Home Assistant Conference

This isn't a highlight yet, but without a doubt, it is definitely is going to
become one of the highlights of this year: The Home Assistant Conference.

> <a href="/conference"><img src='/images/conference/conference-header.png' alt='Home Assistant Conference header' class='no-shadow'></a>

Reviewer: Can we embed a screenshot of the schedule instead

Author: Yes, we can now 👍  (the screenshot was published after this PR).

Will adjust.

</blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

## Home Assistant Conference

This isn't a highlight yet, but without a doubt, it is definitely is going to
become one of the highlights of this year: The Home Assistant Conference.

Sunday, 13 December, we'll be looking back at last year; but more importantly,
lots of great talks are planned by speakers from our community about all
different kinds of subjects:

<a href="/conference"><img src='/images/conference/schedule.png' alt='Home Assistant Conference Schedule' class='no-shadow'></a>

Above all, some new and exciting stuff is going to be announced as well! So,
ensure you don't miss it.

> [Check out our Home Assistant Conference page on how to join!](/conference)

Reviewer: What about something like:

> Get your $1 ticket or check the Home Assistant Conference page for other ways on how to watch.

Author: Updated
